numpy/core/src/umath/fast_loop_macros.h

```diff
@@ -305,7 +305,7 @@ abs_ptrdiff(char *a, char *b)
  */
 #define IS_OUTPUT_BLOCKABLE_UNARY(esizein, esizeout, vsize) \
     ((steps[0] & (esizein-1)) == 0 && \
-     steps[1] == (esizeout) && llabs(steps[0]) < MAX_STEP_SIZE && \
+     steps[1] == (esizeout) && labs(steps[0]) < MAX_STEP_SIZE && \
```

Comment:
This change seems risky, the function needs to handle `npy_intp`, and long type is untrustworthy for that.

Reply:
`llabs` is new in C99, so should be available.

Reply:
Is there a warning about the comparison?

Reply:
`labs` is used for all the other checks, I think that is incorrect. @seiko2plus Thoughts?

Reply:
There was a warning, which lead to the change.

Reply:
I reverted this and there are no warnings with the latest clang-cl, so I've reverted it.